src/reducers/index.js

```diff
@@ -152,7 +152,7 @@ const slice = createSlice({
     },
     propChange(state, action) {
       const { newProps, oldProps } = action.payload;
-      const oldState = clone(state);
+      const oldState = clone(original(state));
```

Comment:
what does original do

Reply:
`state` is proxy object from immer that points to the actual redux state. 
Iterating over this proxy object is slightly slower than iterating over the original state.

`original(proxyObject)` returns back the original object pointed to by the proxy.
(See [original](https://redux-toolkit.js.org/api/other-exports#original) docs)